.vib/magento/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,57 @@
+const CLICK_DELAY = 1000;
+const GET_DELAY = 1000;
+const BASE_URL = 'http://vmware-magento.my';
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, CLICK_DELAY);
+    });
+  });
+}
+
+for (const command of ['get']) {
```

Comment:
There was a need to slow down Cypress fetching the elements as well as the clicks due to the application having bad performance. 
This was my way of making sure that the conditional testing we added doesn't introduce flakiness.